tests/functional/shutdown/08-now1/flow.cylc

```diff
@@ -14,6 +14,5 @@
         script = cylc__job__wait_cylc_message_started; cylc stop --now "${CYLC_WORKFLOW_ID}"
         [[[events]]]
             started handlers = sleep 10 && echo 'Hello %(id)s %(event)s'
-            succeeded handlers = echo 'Well done %(id)s %(event)s'
```

Comment:
The `succeeded` handler is not guaranteed to run because the `succeeded` message is not guaranteed to be received by the scheduler before it shuts down. So I have removed it from the test.

Reply:
The test says:

```
Test "cylc stop --now" will wait for event handler.
```

So removing this kinda defeats the object of the test.

There is a way to ensure that the succeeded message is recieved however,

```ini
script = """
cylc message -- succeeded
cylc workflow-state "$CYLC_WORKFLOW_ID//$CYLC_TASK_CYCLE_POINT/$CYLC_TASK_NAME:succeeded"
sleep 1  # give the scheduler time to process the message
cylc stop --now "${CYLC_WORKFLOW_ID}
"""
```

Reply:
It's still testing that the long-running `started` handler finishes before shutdown. I'm not sure the `succeeded` handler adds any value to the test.

I tried adding `cylc message -- succeeded` but this results in task `t2` starting, which fails the test

Reply:
> I tried adding cylc message -- succeeded but this results in task t2 starting

We could make that work by using an optional output.

> It's still testing that the long-running started

Gotcha, as long as there is one event handler.

The 10s sleep is a little crude (might be better to poll for the workflow "stopping" state or shutdown message), but that's another matter.